parser/src/main/java/me/vertretungsplan/parser/LegionBoardParser.java

```diff
@@ -272,7 +272,9 @@ private Substitution getSubstitution(JSONObject change, HashMap<String, String>
 		}
 		for (int i = 0; i < courses.length(); i++) {
 			final JSONObject course = courses.getJSONObject(i);
-			classes.add(course.getString("name"));
+			if (course.getString("archived").equals("false")) {
```

Comment:
Isn't this a boolean value in the JSON? (-> if yes, use `getBoolean` instead of `getString`).